common/changes/@adobe/ccweb-add-on-analytics/release-20240117_2024-01-17-10-49.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@adobe/ccweb-add-on-analytics",
+      "comment": "Improvements",
```

Can we provide a better description here and all the other change files?
Like:
```suggestion
      "comment": "Bug fixes and improvements",
```